src/Compilers/CSharp/Test/Syntax/project.json

```diff
@@ -2,6 +2,7 @@
   "dependencies": { },
   "frameworks": {
     "netstandard1.3": {
+      // xunit isn't compat with netstandard
```

Comment:
Doesn't matter. They're probably not the only tool that will consume project.json, but even if they are, they are perfectly in their right to start throwing errors about it later, and I don't want to hit errors just because we update our `nuget.exe` version.